.github/workflows/mirror.yml

```diff
@@ -0,0 +1,18 @@
+name: Mirror
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Update download mirror
+      run: script/mirror update "${BEFORE_REF}.."
```

Comment:
It would be interesting to run it once with `6b5c8148228bc3f5e58ab9c54c29d0b71be65524..` (the last time TravisCI ran).
That would both test the upload, and upload new releases added since that commit.

Reply:
You're right! I'll try to trigger the action with that ref. However, I have already pre-filled the mirror while testing on my own machine, so the output we should see is that all downloads have already been mirrored.